pretty_printers_tests/src/main.rs

```diff
@@ -0,0 +1,49 @@
+extern crate test_runner;
+extern crate toml;
+
+use std::fs;
+use std::fs::read_dir;
+use std::path::Path;
+
+use test_runner::{LLDBConfig, LLDBTestRunner};
+
+fn main() -> Result<(), ()> {
+    let settings_file =
+        if cfg!(target_os = "linux") {
```

Comment:
Not sure but maybe separate cfg functions for all supported os are better?
Something like:
```rust
#[cfg(target_os = "linux")]
fn settings() -> &str {
    "Settings_linux.toml"
}
```

Reply:
What do you think about this?
```
#[cfg(target_os = "linux")]
static SETTINGS: &str = "Settings_linux.toml";
#[cfg(target_os = "macos")]
static SETTINGS: &str = "Settings_macos.toml";
#[cfg(all(not(target_os = "linux"), not(target_os = "macos")))]
panic!("Unsupported platform");

fn main() -> Result<(), ()> {
    let settings = fs::read_to_string(SETTINGS).expect(&format!("Cannot read {}", SETTINGS));
    let lldb_config: LLDBConfig = toml::from_str(&settings).expect(&format!("Invalid {}", SETTINGS));
```

Reply:
@ortem generally, looks ok
But it seems that `SETTINGS` can be conts